source/assets/js/helpers/module.js

```diff
@@ -12,7 +12,7 @@ class EstaticoModule {
 	* @param  {object} options - The options passed as data attribute in the Module
 	*/
 	constructor($element, _defaultData, _defaultOptions, data, options) {
-		this.name = this.constructor.name.toLowerCase();
+		this.name = this.constructor.name;
```

Comment:
Ah, got it. Sorry, I only thought of `data-init`. (We use a different data binding, so I forgot about that.)

Don't have a good solution in mind right now. Will think about it